src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -329,51 +339,57 @@ public GHPullRequestReviewComment createReviewComment(String body, String sha, S
                 .to(getApiRoute() + "/comments", GHPullRequestReviewComment.class).wrapUp(this);
     }
 
+    /**
+     * Create a reply to the current comment.
+     */
+    public GHPullRequestReviewComment createReviewCommentReply(GHPullRequestReviewComment comment, String body) throws IOException {
+        return new Requester(owner.root).method("POST")
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)